Herbert/Framework/Enqueue.php

```diff
@@ -66,13 +66,15 @@ public function buildInclude($attrs, $footer)
 //            $attrs['src'] = ltrim($attrs['src'], '/');
 //        }
 
+        if (!isset($attrs['dep'])) $attrs['dep'] = [];
+
         if (pathinfo($attrs['src'], PATHINFO_EXTENSION) === 'css')
         {
             wp_enqueue_style($attrs['as'], $attrs['src']);
         }
         else
         {
-            wp_enqueue_script($attrs['as'], $attrs['src'], [], false, $footer);
+            wp_enqueue_script($attrs['as'], $attrs['src'], $attrs['dep'], false, $footer);
```

Comment:
I'd definitely prepend the 3rd arg with `(array)` to allow for a single string dep to be added.